src/SFML/Window/CMakeLists.txt

```diff
@@ -285,8 +285,8 @@ if(SFML_OS_LINUX OR SFML_OS_FREEBSD OR SFML_OS_OPENBSD OR SFML_OS_NETBSD)
         sfml_find_package(GBM INCLUDE "GBM_INCLUDE_DIR" LINK "GBM_LIBRARY")
         target_link_libraries(sfml-window PRIVATE drm gbm EGL)
     else()
-        sfml_find_package(X11 INCLUDE "X11_INCLUDE_DIR" LINK "X11_X11_LIB" "X11_Xrandr_LIB" "X11_Xcursor_LIB")
-        target_link_libraries(sfml-window PRIVATE X11)
+        find_package(X11 REQUIRED)
```

Comment:
```suggestion
        find_package(X11 REQUIRED COMPONENTS Xcursor Xrandr)
```

Reply:
Is it possible that the X11 package may not contain these components or is this mostly about expressiveness? My impression of [FindX11.cmake](https://cmake.org/cmake/help/latest/module/FindX11.html) was that all targets were always available.

Reply:
Not all targets are always available. For example, some need a specific apt package (see [ci.yml](https://github.com/SFML/SFML/blob/eb162df723c178c330f7ec33dc6a288267f2afe3/.github/workflows/ci.yml#L60)). I tested configuring on my system and a great proportion of cache variables from FindX11.cmake end with NOTFOUND.

Reply:
In that case I'm cool with us explicitly listing components.